aspnetcore/tutorials/first-mvc-app/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Create a web app with ASP.NET Core MVC
 author: rick-anderson
-description: See the table of contents for an introduction to ASP.NET Core MVC.
+description: This series of tutorials explains the basics of building a web app with ASP.NET Core MVC.
```

Comment:
There's no content here, this is click bait. I'd like to delete this file.

Reply:
Removed Overview (index.md)
Added original Overview URL to redirect which pointes to start-mvc.md.
Updated links for start-mvc on ASP.NET Core Landing page.